web/src/components/AppBanner.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <v-banner
+    app
+    color="blue"
+    dark
+    icon="mdi-information"
+    class="ma-4"
+  >
+    <span class="title">Anouncement: </span>The NMDC will be replacing legacy identifiers with
+    <a href="https://microbiomedata.github.io/nmdc-schema/identifiers/#ids-minted-for-use-within-nmdc">
+      NMDC persistent identifiers
```

Comment:
When rendered, this link ends with an underlined space.

![image](https://github.com/microbiomedata/nmdc-server/assets/134325062/84f6e3a8-4e1e-41df-becd-ed4c90820c88)

Reply:
Fixed in https://github.com/microbiomedata/nmdc-server/pull/1216